### ISSUE TITLE
Training configuration for different models

### ISSUE THREAD

**Issue by justinpinkney** [bryandlee/FreezeG#9]
Hi! Really nice work, the examples are great. I was wondering if you had listed the training configuration somewhere, specifically which resolution layers your froze? I'm also wondering if you did any experiments with freezing different subsets of the modules within each conv?

For context I worked on the "post training" version of this method, i.e. swapping the trained layers form one model into another, see: https://arxiv.org/abs/2010.05334 When I was doing this work I thought that doing something like your freezeG approach might be a more direct way of getting the same effect. I tried a couple of runs myself at the time, but could never get good results (the network failed to learn to generate good images). Curious if you have any insight of things you tried which didn't work?

**Comment by bryandlee** [comment]
Hi! Thanks for your interest. Unfortunately, I did not keep the full configuration of the experiments, but some important parameters that I used are

* batch=8, use ADA
* face2met: finetune_loc=2, iter=1000
* cat2wild: finetune_loc=3, iter=3000

The numbers are not tuned, I simply chose fewer fine-tuning layers and iterations when the source and target domain is close. Freezing different submodules of the model rather than the entire styled-conv layer didn't work well in my case.

I actually tried the layer swapping method similar to yours as well but failed to make it work nicely. In my experience, the problem was the misalignment of input domains for the layers from different models, and I guess the smooth interpolation technique is crucial. (I really like the Disney-style cartoonification results btw)
